middleware/markdown/summary/render.go

```diff
@@ -9,23 +9,26 @@ import (
 // Ensure we implement the Blackfriday Markdown Renderer interface
 var _ blackfriday.Renderer = (*Renderer)(nil)
 
+// Renderer is a plain-text Markdown renderer that implements the
+// blackfriday.Renderer interface.  Many of the required methods are
+// stubs with no output.
 type Renderer struct{}
```

Comment:
Although I've no idea where else we use this, and if anyone utilizes this in his/her plugins, I feel we should unexport this and see if anyone complains.